bucket/gdb.json

```
@@ -1,5 +1,6 @@
 {
     "homepage": "http://tdm-gcc.tdragon.net/",
+    "description": "GDB, the GNU Project debugger, allows you to see what is going on `inside' another program while it executes -- or what another program was doing at the moment it crashed. (TDM port)",
```

```suggestion
    "description": "GNU Project debugger that allows you to see what is going on `inside' another program while it executes -- or what another program was doing at the moment it crashed. (TDM port)",
```

---

Small change for @niheaven suggestion to keep same quote

```suggestion
    "description": "GNU Project debugger that allows to see what is going on 'inside' another program while it executes -- or what another program was doing at the moment it crashed. (TDM port)"
```